assets/bpmn-js.css

```diff
@@ -113,4 +113,61 @@
 
 .selected .bjs-drilldown-empty {
   display: inherit;
+}
+
+/* COLOR PICKER */
+
+.bpmn-color-picker .djs-popup-header {
```

Comment:
We should namespace _all_ classes (`bjs-***`).

Reply:
`bpmn-color-picker` is derived from `popupMenu.registerProvider('bpmn-color-picker', _)`. I name it like this following the convention of `bpmn-replace`

Reply:
You could call it `bjs-color-picker` or whatever, but we should always namespace new additions to our CSS.